header.go

```diff
@@ -681,6 +681,8 @@ func (h *RequestHeader) RequestURI() []byte {
 	requestURI := h.requestURI
 	if len(requestURI) == 0 {
 		requestURI = strSlash
+	} else if requestURI[0] == '?' {
+		requestURI = append(strSlash, requestURI...)
```

Comment:
This would cause an allocation, which we always try to avoid in fasthttp. I'm wondering if it isn't better to add the `/` here when needed: https://github.com/valyala/fasthttp/blob/699af40ee434bdefa692ff30da20d428d3b6fceb/header.go#L2815

Reply:
Ah, great point. I've updated the change to modify the request URI when set through either the `SetRequestURI` or `SetRequestURIBytes` functions.